App.js

```diff
@@ -12,7 +12,7 @@ const AppWithFonts = () => {
   const { trackAppStart } = useMatomo();
 
   trackAppStart();
-
+    
```

Comment:
these spaces should be reverted

Reply:
done in e2f9b849f1157cec42932f4fc81cf4ce13f20fc8